### ISSUE TITLE
Documentation interrupted due to inaccessibility of the bostom dataset

### ISSUE THREAD

**Issue by thibaultcordier** [scikit-learn-contrib/MAPIE#460]
**Is your documentation request related to a problem? Please describe.**
Documentation construction is interrupted due to the inaccessibility of the bostom dataset, which fails the current PR.

**Describe the solution you'd like**
Switch from the bostom dataset to the toy dataset.
